### PR TITLE
Feat/clientUrls

### PR DESCRIPTION
**Pull request by Sumitabhbiswas**

### CODE REVIEW

**Review by Surendhar1997 — APPROVED** [review]
LGTM

**Review by serialcoder13 — CHANGES_REQUESTED** [review]
Please make those suggested changes.

File: security/src/main/java/com/fincity/security/dao/ClientUrlDAO.java [hunk 1]
```diff
@@ -32,17 +37,31 @@ protected Field<ULong> getClientIDField() {
...
+	}
+
+	
+	public Mono<List<String>> getClientUrlsBasedOnAppAndClient(String appCode, ULong clientId) {
+
+		List<Condition> conds = new ArrayList<>();
+
+		conds.add(SECURITY_CLIENT_URL.APP_CODE.eq(appCode));
+
+		if (!clientId.equals(null))
```
Comment (serialcoder13) [inline security/src/main/java/com/fincity/security/dao/ClientUrlDAO.java:60]: you should check clientId != null. If the clientId is null you can't invoke equals.

**Review by serialcoder13 — CHANGES_REQUESTED** [review]
Please update the changes

**Review by serialcoder13 — APPROVED** [review]
LGTM

**Pull request merged**